compiler/src/dotty/tools/dotc/typer/Checking.scala

```diff
@@ -671,30 +671,32 @@ trait Checking {
       report.error(ex"$cls cannot be instantiated since it${rstatus.msg}", pos)
   }
 
-  /** Check that pattern `pat` is irrefutable for scrutinee tye `pt`.
-   *  This means `pat` is either marked @unchecked or `pt` conforms to the
+  /** Check that pattern `pat` is irrefutable for scrutinee tye `sel.tpe`.
```

Comment:
There is a small type: "tye" -> "type"
```suggestion
  /** Check that pattern `pat` is irrefutable for scrutinee type `sel.tpe`.
```